app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -428,7 +437,7 @@ public void onBindViewHolder(Presenter.ViewHolder viewHolder, Object item) {
             }
         }
 
-        BitmapDrawable blurHashDrawable = null;
+        String blurHash = "";
```

Comment:
Wouldn't it make more sense to use a null value here and allow the null value in the `blurHashPlaceholder` extension function?

Reply:
Probably. I started down that route initially and then changed to avoid needing null handling on the Kotlin side... I've changed it back now. :smile: